### ISSUE TITLE
Help with configuration

### ISSUE THREAD

**Issue by faz17** [agittins/bermuda#396]
Hello,
I'm pretty new to Home assistant and Bermuda.
I'm trying to set up a simple presence system with bluetooth that tells me if my phone is home or not. I have an android with the Ha app installed and I have activated the BLE transmitter. At the moment I'm not interested in presence inside the single room. As a scanner I have set up a shelly and the bluetooth of the raspberry that are in the same room. The problem is that when I'm close to the scanner it continues to switch from unavailable to detected. If I move away even less than 10 meters the phone is not detected at all.
I don't know if I'm doing something wrong in the phone configurations or in Bermuda.

**Comment by agittins** [comment]
Howdy! Please upload a "download diagnostics" from Bermuda, then I can take a look and see what's going on.

Does it behave the same if you are close to the Shelly or the Pi, or is it mainly the Pi that is inconsistent?

**Comment by faz17** [comment]
Hello! Thank for your help and work.
the shelly and the Pi are 20 cm apart from each other...so seems the same.
I attach the diagnostic. I have made some test... so maybe you see different values
[config_entry-bermuda-01JD28JHY3NW2H3QR72X5WC9TS.json](https://github.com/user-attachments/files/17843992/config_entry-bermuda-01JD28JHY3NW2H3QR72X5WC9TS.json)

**Comment by agittins** [comment]
Thanks for sending that through. I can see that the pi's bluetooth adaptor seems to be working well. It can see other bluetooth devices in (or near) your home such as the Samsung TV, the Shelly1 and the Fridge.

However... there is no Shelly device acting as a bluetooth proxy that Bermuda can see. There is a "Shelly Plus 1" with a wifi MAC address ending in D4, but it doesn't show up as a proxy (I can see it because the pi is picking up its bluetooth advertisements - so it's **sending** advertisements, but not **receving/proxying** them).

I'd suggest for the Shelly:
- make sure that it's updated to the latest firmware
- remove any scripts you might have added to it manually
- in the HomeAssistant Shelly integration, enable the bluetooth proxy for the Shelly (this will install the correct script onto the device).

That *should* get the Shelly working as a proxy.

As for the phone....

The only "configured devices" I can see are for two iBeacons, one starting with `5FDE` the other starting `1086`, and both having a major_id of `100` and a minor_id of `40004`. Neither of those have been "seen" yet by the pi.

However there is another iBeacon, starting with `1BC5`, major 225 minor 22510, and *that* one is being picked up, but the signal is not as reliable as the fridge or tv. It has big gaps between receiving adverts, sometimes 7 seconds, 11, or 15 seconds. But you don't seem to have this iBeacon in your `configured devices` list in Bermuda.

Can you send a screenshot of your phone's iBeacon app settings? Are you using the HA Companion app on Android, or something else?

To sum up:
- The Shelly doesn't seem to be acting as a proxy at all
- Your **configured** devices don't appear to be present, but the fridge and tv are sending adverts just fine
- There is another iBeacon which looks close (the 1BC5 one) but we aren't seeing its advertisements reliably.

If you can check the above, and send through a screenshot of your app settings we'll see what we can work out from there! 👍🏼

**Comment by faz17** [comment]
Thanks for your reply.
Yes i have a shelly 1 but stopped It to act as proxy for make some test and forgot to reenable It. 
Now i have reenable It and i attach a new diagnostic.
5FDE and 1086 are my two phone. I'm using on both android HA companion app.
This Is my settings
![Screenshot_20241121_190154_Home Assistant](https://github.com/user-attachments/assets/1d48fff5-34ef-4735-94b0-00fd37505fd6)
![Screenshot_20241121_190143_Home Assistant](https://github.com/user-attachments/assets/1a30edd1-5ce4-4bfb-964c-9641553ef182)
[config_entry-bermuda-01JD28JHY3NW2H3QR72X5WC9TS.json](https://github.com/user-attachments/files/17850310/config_entry-bermuda-01JD28JHY3NW2H3QR72X5WC9TS.json)

**Comment by faz17** [comment]
I add some information....I tried to increase the ble signal strength on the HA app.

For the 1086 device I put medium and it seems to work correctly.

For the 5fde device (samsung zfold6) I had moments in which it was not detected.

Both phones were on charging next to each other.
At this point it seems like a problem with the phone (5fde).
I increased the bermuda settings for the not home state to 120
![immagine](https://github.com/user-attachments/assets/62f5504a-d6e7-4e10-8dab-cda765cc0530)
